.github/workflows/ci.yml

```
@@ -19,77 +19,98 @@
   lint:
     runs-on: ubuntu-latest
-    if: "${{ contains(github.event.pull_request.labels.*.name, 'PR status: code ready') }}"
-    timeout-minutes: 15
+    if: "${{ github.event.label.name == 'PR status: code ready' }}"
+    timeout-minutes: 10
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+      - uses: actions/setup-node@v3
```

actionlint (1.7.7)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)